dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -241,10 +248,26 @@ DataInterfaceIIDM::getBusName(const std::string& componentName, const std::strin
         busName = line->getBusInterface()->getID();
         break;
       }
-      case ComponentInterface::TWO_WTFO:  // @todo with @NODE1@ , @NODE2@
+      case ComponentInterface::TWO_WTFO: {
+        shared_ptr<TwoWTransformerInterface> twoTransformer = dynamic_pointer_cast<TwoWTransformerInterface>(component);
+        if (labelNode == "@NODE1@") {
+          busName = twoTransformer->getBusInterface1()->getID();
+        } else if (labelNode == "@NODE2@") {
+          busName = twoTransformer->getBusInterface2()->getID();
+        }
         break;
-      case ComponentInterface::THREE_WTFO:  // @todo with @NODE1@, @NODE2@
+      }
+      case ComponentInterface::THREE_WTFO: {
+        shared_ptr<ThreeWTransformerInterface> threeTransformer = dynamic_pointer_cast<ThreeWTransformerInterface>(component);
+        if (labelNode == "@NODE1@") {
+          busName = threeTransformer->getBusInterface1()->getID();
+        } else if (labelNode == "@NODE2@") {
+          busName = threeTransformer->getBusInterface2()->getID();
+        } else if (labelNode == "@NODE3@") {
```

Comment:
Please modify the doxygen doc of this method to specify that @NODE3@ is a possible value